docs-source/content/userguide/managing-domains/configoverrides/_index.md

```diff
@@ -366,6 +371,14 @@ Incorrectly formatted override files may be accepted without warnings or errors
       * This directory, or a subdirectory within this directory, should contain each of your custom situational configuration files.
       * If it doesn't, then this likely indicates your domain resource `configOverrides` was not set to match your custom override configuration map name, or that your custom override configuration map does not contain your override files.
 
+* If the Administration Server pod does start but fails to reach ready state:
```

Comment:
Also, please move this bullet up above the previous bullet, so it appears just after the other 'does start' bullet.

Reply:
Do you mean move this bullet to before line 370?

Reply:
Yes.

Reply:
Done